src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -91,3 +91,11 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
   }
 }
 </script>
+
+<style lang="scss" scoped>
```

Comment:
Moved here, as close to the sub-component as possible. (Was prev in parent component.)

I tested and it looks good in all filings that use this component.